tests/e2e/utils.js

```diff
@@ -52,6 +44,23 @@ export function getExtensionElement(id) {
   return $(`>>>[data-qa="${id}"]`);
 }
 
+export async function waitForIdleBackgroundTasks() {
+  const protocol = browser.isChromium ? 'chrome-extension' : 'moz-extension';
```

Comment:
wasn't protocol defined somewhere else already? in config?
Maybe it would be possible to attach some data to the browser object?

Reply:
Yes, but above, not in the config.